### DISCUSSION TITLE
How to Use Number to Extend Secletion

### DISCUSSION THREAD

**Discussion by drcxd** [meow-edit/meow#305] [General]
After press `e` in normal mode, there will be 20 numbers displayed on the screen. I know I could use `0-9` to quickly expand the selection region to one of the ten locations the numbers indicate. I am wondering how could I access the other ten locations also labeled from `1` to `0`? If I can't access them, then why they are displayed?

**Comment by Linerre** [comment]
I don't remember if there is a direct way to jump to the number in the next range from current position, but this is usually what I do:
1. press `e` 
2. press `0` to go to the furthest (the last position) of the current selection range
3. press the number, say, `6` in the next range to jump to it
4. keep doing this until you stop the desired position. (but this is rare and not recommended)

**Comment by fbergroth** [comment]
Press the target digit, and keep hitting `0` until you reach it.
